UN-OCHA/response-site
# RWR-260 Attempt to bring RWR, CD, RW styles together

Using the CD typography changes in https://github.com/UN-OCHA/common_design/pull/342 and `cd-block-title` component for consistency, with some changes directly in the sub theme to align RWR styles more closely to RW styles.

Screenshot comparisons with current site, and Reliefweb site on RW-260

## To test

1. Checkout this branch
2. In `html/themes/contrib` delete the `common_design` base theme (created when composer is run and non-writable) and clone it again from https://github.com/UN-OCHA/common_design. Then checkout `CD-411-RW` branch of the base theme.
3. Clear cache
4. The base theme branch and the changes from this PR for the sub theme combined should reflect what is captured in the screenshots on RW-260
